app/views/orders/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>New</h1>
```

Comment:
L'application doit être en anglais ?

Ici il faut mettre le texte dans un fichier contenant les `locales` de la langue souhaité (présent dans `config/locales/fr.yml`

Exemple de format du .yml:

```yml
en:
  order: 
    new: 'Créer une facture'
```
Qui permet d'appeler le contenu dans les vues de cette manière:

```ruby
<%= t('order.new') %>
```
Attention, il faut définir la langue par défaut dans `config/application.rb` (google it !)